### ISSUE TITLE
Different exceptions for SC 2.5.5 Target Size(Enhanced) and SC 2.5.8 Target Size (Minimum)

### ISSUE THREAD

**Issue by getaccessibility** [w3c/wcag#3714]
Though conceptually the intent of SC 2.5.5 Target Size(Enhanced) and SC 2.5.8 Target Size (Minimum) is same, they have different exceptions (in terms of requirements as well as wordings). Here is the summary for quick reference:

Difference #1: (missing exception for 2.5.5)
The exception requires appropriate spacing to avoid accidental clicks but does not explicitly call for a larger clickable area. Therefore, this could be a reason for not including it in the 2.5.5 SC, as it adheres to AAA standards, which are somewhat more stringent.
Exception: 
Spacing: Undersized targets (those less than 24 by 24 CSS pixels) are positioned so that if a 24 CSS pixel diameter circle is centered on the bounding box of each, the circles do not intersect another target or the circle for another undersized target;

Difference #2: 
Though meaning is same, the wording is different for ‘Equivalent’ exception.
2.5.8 - Equivalent: The function can be achieved through a different control on the same page that meets this criterion;
2.5.5 - Equivalent: The target is available through an equivalent link or control on the same page that is at least 44 by 44 CSS pixels;
Proposal : Can we use the same wording for exceptions for both of these success criteria to ensure consistency?

Difference #3:
The intent is same for the exception ‘Inline’ but for 2.5.8, it is elaborated to explicitly/include line-height constraint callout.
2.5.5 - Inline: The target is in a sentence or its size is otherwise constrained by the line-height of non-target text;
2.5.8 - Inline - The target is in a sentence or block of text;
Proposal : Can we use the same wording for exceptions for both of these success criteria to ensure consistency?

Difference #4: 
Only difference for ‘Essential’ rule is calling out ‘Legal’ requirement for 2.5.8 which is missing for 2.5.5
2.5.5 - Essential: A particular presentation of the target is essential or is legally required for the information being conveyed.
2.5.8 - Essential: A particular presentation of the target is essential to the information being conveyed.
Porposal : Please can we explain if 'Legal requirement' is an exception or not for 2.5.5 in the "Understanding Text Size(Enhanced)" documentation? If it is an exception for 2.5.5 as well, then can we use the same wording for exceptions for both of these success criteria to ensure consistency?

**Comment by alastc** [comment]
**Difference 1:** This is intentional as the aim is slightly different, and at the AA level it is more flexible. (The aim for 2.5.8 is to prevent mistaken clicks/taps, but with 2.5.5 the aim is correct click/taps.)

**Difference 2:** At the AAA level there is none of the exceptions apply for the equivalent control, whereas in the AA version the exceptions do apply. This is a meaningful and intentional difference to allow more flexibility at the AA level.

**Difference 3:** The wording in the newer SC is better, it would be nice to update the previous SC, however it isn't a high priority change. It also would be a change, which means there is quite a bit of process to go through. 

**Difference 4:** Similar to 2, the newer one is slightly better, but also aiming to be more flexible. 

Overall, we did go through a process of aligning the SC as best we could within the constraints. I don't think there will be an appetite for changing this, particularly as one is at the AAA level. If you are looking at doing AAA (which isn't generally required) then I think you'd take the overall "make targets bigger" approach and be less concerned by the exceptions.

**Comment by getaccessibility** [comment]
Thank you @alastc for looking in to it. Appreciate your detailed comments. 

Difference 1: Totally agree and understand that the exception is flexible for AA and strict for AAA. 

Difference 2: I can see this exception for AAA as well. 
Equivalant - The target is available through an equivalent link or control on the same page that is at least 44 by 44 CSS pixels;

Difference 4: So for AAA, 'legal information is not an exception where as for AA, 'legal information' is an exception?
Please suggest if I am understanding it differently.

**Comment by Jym77** [comment]
> **Difference 2:** At the AAA level none of the exceptions apply for the equivalent control, whereas in the AA version the exceptions do apply. This is a meaningful and intentional difference to allow more flexibility at the AA level.

I was looking for this clarification 😊

Then, we may need to update [Understanding 2.5.8](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html):
> **Equivalent**: In cases where a target does not have a size equivalent to 24 by 24 CSS pixels, but there is another control that can achieve the underlying function that does meet the minimum size, the target can be excepted based on the "Equivalent" exception.

This explanation suggests that the equivalent control needs to be 24×24px, not that it can be spaced enough, or inline, …
(interpreting "meet the minimum size" as "24×24px", rather than "meet the minimum size SC", which is maybe my mistake here)

**Comment by scottaohara** [comment]
i can see why someone, reading this revised explanation of equivalent could come away with this misunderstanding.  the normative text to explain equivalent is clear.  i'll make a quick update PR
